.eslintrc.yml

```diff
@@ -0,0 +1,175 @@
+rules:
```

Comment:
This should be using [the shared probo rules](https://github.com/ProboCI/eslint-config-probo) see the [probo implementation](https://github.com/ProboCI/probo/blob/master/.eslintrc).